### ISSUE TITLE
Problem: not possible to retrieve app version via RPC

### ISSUE THREAD

**Issue by tomtau** [crypto-org-chain/chain-main#362]
Not sure if I missed it anywhere, but I don't see a way to retrieve the version?
Normally, it could  e.g. be a part of a response in abci_info, but that doesn't seem to be returned at the moment:
```
{
  "jsonrpc": "2.0",
  "id": -1,
  "result": {
    "response": {
      "data": "chain-maind",
      "last_block_height": "...",
      "last_block_app_hash": "..."
    }
  }
}
```

also one thing to try is if the version is non-zero, everything still works -- we previously had some "fun" when we bumped the app version to 1: https://github.com/crypto-com/thaler/issues/1527 :D

**Comment by tomtau** [comment]
From Discord:
```
tomtau01/26/2021
Any reason why baseapp's SetAppVersion doesn't set/return that version in abci info response? Is it expected to be always set/overwritten by applications?
 
aleksbez | Interchain01/26/2021
Good question
[8:35 PM]
Probably meant to be overridden
[8:36 PM]
The version gets set on EndBlock which you have to implement anyway
[8:36 PM]
But I think you’re right
```

**Comment by linfeng-crypto** [comment]
starting

**Comment by linfeng-crypto** [comment]
Somebody already submited a [PR](https://github.com/cosmos/cosmos-sdk/pull/8897) for cosmos-sdk

**Comment by regisd** [comment]
FYI https://github.com/cosmos/cosmos-sdk/pull/8897 has been merged.

**Comment by tomtau** [comment]
yeah, this can probably be closed once the repo upgrade to 0.43  -- ref https://github.com/crypto-org-chain/chain-main/pull/502
